worker/jobs/move-contact/move-contact-job.worker.ts

```diff
@@ -0,0 +1,43 @@
+import { Job } from 'bullmq';
+import { BullMQWorker } from '../../worker';
+import { MoveContactJobProcessor } from './move-contact-job.processor';
+import { queueManager } from '../../../shared/queues';
+
+export interface MoveContactParams {
+  name: string;
+  parentId: string;
+  contactId: string;
+  sessionToken: string;
+  instanceUrl: string;
+}
+
+export class MoveContactJobWorker extends BullMQWorker {
+  
+  DELAY_IN_DAYS = 3;
+
+  constructor(queueName: string, processor = new MoveContactJobProcessor()) {
+    super(queueName, processor);
+  }
+
+  async canProcess(job: Job): Promise<boolean> {
+    // TODO: Implement logic here
```

Comment:
i think to go-live with this, we need to make a request to the monitoring endpoint (https://nairobi.echis.go.ke/api/v2/monitoring) and confirm that `sentinel.backlog` is less than 1000. 
this wil ensure we don't take down the server by moving contacts too quickly

it should postpone probably 1 hour? something like that?